CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # Radicle Planning Boards Change Log
 
+## **v0.2.0** - Jul 2nd 2024
+
+### 🚀 Enhancements
+
+- Add support for filtering via a text field
```

Comment:
We could and perhaps should say more about this and its subfeatures.

For example that it searches titles and ids, that it does partial string matching, that it highlights on the card exactly which part of it matched, the keyboard shortcut, the escape key to unfocus, and anything else I'm forgetting.